.mci.yml

```diff
@@ -848,7 +879,7 @@ buildvariants:
 
     # Test that we actually support building against the minium required C version claimed.
     - name: ubuntu1804-min-libmongoc
-      display_name: "Ubuntu 18.04 with minimum libmongoc (MongoDB 4.4)"
+      display_name: "Ubuntu 18.04 with minimum libmongoc (MongoDB Latest)"
```

Comment:
I'm not sure why the description said 4.4 but it uses the latest. I'm not sure if we actually wanted to use the latest or 4.4 though.